binplace.targets

```diff
@@ -22,27 +22,37 @@
     <BinPlaceStuffDependsOn Condition="'$(BinPlaceRuntime)' == 'true'">$(BinPlaceStuffDependsOn);BinPlaceRuntimeAssembly</BinPlaceStuffDependsOn>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Yeah, could be.  I was thinking this might be better than sprinkling the use of _bc_* all over.